src/app/components/navbar/navbar.component.html

```diff
@@ -1,8 +1,8 @@
 
     <nav class="navbar navbar-expand-lg navbar-light mainnav" aria-label="Site navigation menu">
 
-          <a class="navbar-brand logoRev" href="#">
-              <img class="imageLogo" src="../assets/revature-logo-600x219.png" routerLink="/" alt="The Revature logo.">
+          <a class="navbar-brand logoRev" routerLink={{link}}>
```

Comment:
Should this instead be [routerLink]="link" instead?

Reply:
@grturner

Reply:
Since being logged-in reassigns navbar's `link` to the appropriate endpoint, property binding vs string interpolation accomplishes the exact same thing AFAICT.

Reply:
That is correct, but property binding is preferred in this context. Perhaps that is just my preference, but I usually find that string interpolation is used for innerHTML and property binding is used for attribute values.